src/components/AboutUsSection.astro

```diff
@@ -0,0 +1,32 @@
+---
+import { Image } from "astro:assets";
+
+import CommunityCard from "./CommunityCard.astro";
+
+import imagenInfoSection1 from "../assets/images/imagen-info-section-1.png";
+import logoGdg from "../assets/images/logo-gdg.png";
+import logoWtm from "../assets/images/logo-wtm.png";
+---
+
+<section class="container mx-auto grid grid-cols-1 gap-16 px-3 py-16 md:px-0">
+  <Image class="mx-auto" src={imagenInfoSection1} alt="Devfest 2018" />
```

Comment:
It should be Devfest 2023 instead 2018

Reply:
En realidad está bien, porque la imagen es del devfest de ese año, y el alt text hace referencia a una descripción de la imagen